manifests/base/role.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: moneypenny
+rules:
+- apiGroups: [""]
+  resources: [ "pods", "configmaps"]
+  verbs: ["get", "list", "create", "watch", "delete"]
```

Comment:
I can break it into two things, but I do need to be able to at least get/create/delete configmaps.

Reply:
Well, create/delete anyway.

Reply:
This is all constrained to the `moneypenny` namespace, so I think it's fine. I would be pickier about the exact permissions if it were a `ClusterRole`, but since all that moneypenny could mess with is its own configmaps, I have no security worries here.